app/src/main/java/org/thunderdog/challegram/Log.java

```diff
@@ -407,7 +408,7 @@ private static void notifyLogFilesAltered () {
     TAG_YOUTUBE,
     TAG_CAMERA,
     TAG_EMOJI,
-
+    TAG_LANGUAGE,
```

Comment:
From the 361 line of this file:
> // Don't forget to duplicate new values in log.h

`app/jni/log.h` and `app/jni/log.cpp` files must be updated when adding new log tags.

Also `version.jni` must be bumped after making some changes to native library.